werbies.json

```diff
@@ -957,7 +957,7 @@
       "https://i.etsystatic.com/17523062/r/il/7a2ce5/2828485403/il_570xN.2828485403_8fbl.jpg",
       "https://images-wixmp-ed30a86b8c4ca887773594c2.wixmp.com/f/43d7c22d-6b9c-4a70-86b4-4d5c68ed816c/deivzis-8993f4db-5cbc-48cd-b84f-082fa6c95577.png/v1/fill/w_558,h_350,strp/zagreus_and_cerberus_by_seyumei_deivzis-350t.png?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJ1cm46YXBwOjdlMGQxODg5ODIyNjQzNzNhNWYwZDQxNWVhMGQyNmUwIiwiaXNzIjoidXJuOmFwcDo3ZTBkMTg4OTgyMjY0MzczYTVmMGQ0MTVlYTBkMjZlMCIsIm9iaiI6W1t7ImhlaWdodCI6Ijw9ODA0IiwicGF0aCI6IlwvZlwvNDNkN2MyMmQtNmI5Yy00YTcwLTg2YjQtNGQ1YzY4ZWQ4MTZjXC9kZWl2emlzLTg5OTNmNGRiLTVjYmMtNDhjZC1iODRmLTA4MmZhNmM5NTU3Ny5wbmciLCJ3aWR0aCI6Ijw9MTI4MCJ9XV0sImF1ZCI6WyJ1cm46c2VydmljZTppbWFnZS5vcGVyYXRpb25zIl19.G2j_FBPoO5ZZmsY9nq53kH1xDdysEQ6WHatqKSh_XoI"
     ],
-    "shift4shop_id": 1
+    "shift4shop_id": 49
```

Comment:
is this an intended change?

Reply:
yeah, we never added the hades wax melts